core/src/main/resources/filodb-defaults.conf

```diff
@@ -249,9 +249,9 @@ filodb {
     #      = 5MB per cass partition. Goal is to keep it under 10MB.
     pk-by-updated-time-table-num-splits = 200
 
-    # TTL for rows in partitionkeys_by_update_time table. Ensure it is long enough to ensure that
+    # TTL for rows in ingestion time index tables. Ensure it is long enough to ensure that
     # downsampler or repair migration will complete in that time
-    pk-by-updated-time-table-ttl = 3 days
+    write-time-index-ttl = 7 days
```

Comment:
Is it a good idea to move this as well as part of dataset source store config along with `disk-time-to-live`:
https://github.com/filodb/FiloDB/blob/develop/conf/timeseries-dev-source.conf#L32

Otherwise multiple conf needs to be updated.

Reply:
`ingestion-time-index-retention` is unused config. Removed. Thanks for the catch!